tests/core.py

```diff
@@ -2193,13 +2193,6 @@ def test_get_connections_env_var(self):
         assert conns[0].password == 'password'
         assert conns[0].port == 5432
 
-    def test_get_connections_db(self):
-        conns = BaseHook.get_connections(conn_id='airflow_db')
-        assert len(conns) == 1
-        assert conns[0].host == 'localhost'
-        assert conns[0].schema == 'airflow'
-        assert conns[0].login == 'root'
```

Comment:
I did remove this test because it depends on the ENV of the testing. For example, on travis with postgres this will be host `postgres` instead of localhost. This specific connection will tested on a lot of places anyway so a explicit test here it not needed anyway.